applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+			          group must follow the closing parentheses. That is, all element and group 
+			          multipliers are assumed to be printed as subscripted numbers.
+			        * Unless the elements are ordered in a manner that corresponds to their chemical 
+			          structure, the order of the elements within any group or moiety depends on 
+			          whether or not carbon is present.
+			        * If carbon is present, the order should be: 
+
+			          - C, then H, then the other elements in alphabetical order of their symbol. 
+			          - If carbon is not present, the elements are listed purely in alphabetic order of their symbol. 
+
+			        * This is the *Hill* system used by Chemical Abstracts.
+		        </doc>
+	        </field>
+            <field name="temperature" type="NX_FLOAT" units="NX_TEMPERATURE" minOccurs="0" maxOccurs="1">
+		        <doc>Sample temperature. This could be a scanned variable</doc>
+		        <dimensions rank="anyRank">
+		                	<dim index="1" value="n_Temp"/><!-- could be any length -->
+		        </dimensions>
+	        </field>
+            <field name="stress_field" type="NX_FLOAT" units="NX_ANY" minOccurs="0" maxOccurs="1">
+		        <doc>Applied external stress field</doc>
+		        <dimensions>
+			            <dim index="1" value="n_sField"/><!-- could be any length -->
+		        </dimensions>
+                <attribute name="direction">
+        	        <enumeration>
+        		        <item value="x"/>
+        		        <item value="y"/>
+        		        <item value="z"/>
+        	        </enumeration>
+                </attribute>
+	        </field>
+            <group name="gauge_volume" type="NXparameters" minOccurs="0" maxOccurs="1">
+                <doc> The gauge volume can be described with the following parameters:
+                 .. figure:: stress/GV2.jpg
+                  :width: 70%
+                  :alt: Dimensions of the gauge volume.
+
+                  Parameters describing the gauge volume.
+                </doc>
+                <field name="x" type="NX_FLOAT" units="NX_LENGTH" minOccurs="0" maxOccurs="1">
+                    <doc>
+                        Length of the first diagonal.
+                    </doc>
+                </field>
+                <field name="y" type="NX_FLOAT" units="NX_LENGTH" minOccurs="0" maxOccurs="1">
+                    <doc>
+                        Length of the second diagonal normal to :ref:`x &lt;/NXstress/ENTRY/sample_description/gauge_volume/x-field&gt;`.
+                    </doc>
+                </field>
+                <field name="z" type="NX_FLOAT" units="NX_LENGTH" minOccurs="0" maxOccurs="1">
+                    <doc>
+                        Height of the gauge volume.
+                    </doc>
+                </field>            
+                <field name="depends_on" type="NX_CHAR" minOccurs="0" maxOccurs="1">

```

Comment:
`NXmx` has both `depends_on` and `NXtransformations` in the `NXsample` group level

Reply:
Indeed.